### DISCUSSION TITLE
Help downloading dependencies via metadata.dependencies

### DISCUSSION THREAD

**Discussion by yeachan153** [paketo-buildpacks/feedback#210] [Q&A]
Hi all,

I'm struggling to find documentation about how I can include dependencies via `[[metadata.dependencies]]` as part of the `buildpack.toml`.

I have some jars which I would like to download from maven in `[[metadata.dependencies]]`. Is this possible? Even better, is there some documentation to show me how to do this?

Thanks!

**Comment by dmikusa** [comment]
The buildpacks are not meant to be a general solution for downloading JAR/app dependencies. You should use your build tool of choice to do that. In this case, Maven.
